CONTRIBUTING.md

```diff
@@ -95,8 +95,57 @@ In multi-line bullet point entries, subsequent lines should be indented.
 - Insert a blank line before at-clauses/tags.
 - Favor `{@code foo}` over `<code>foo</code>`.
 - Favor literals (e.g., `{@literal @}`) over HTML entities.
+- New classes should have the JavaDoc that describes the class
+- New classes should have `@since ...` annotation  
 - Use `@since 5.0` instead of `@since 5.0.0`.
-- Do not use `@author` tags. Instead, contributors are listed on [GitHub](https://github.com/junit-team/junit5/graphs/contributors).
+- Do not use `@author` tags. Instead, contributors are listed on [GitHub](https://github.com/junit-team/junit5/graphs/contributors). 
+- Do not use verbs in third person form (use "Get" instead of "Gets" for example). 
```

Comment:
```suggestion
- Do not use verbs in third person form (e.g. use "Discover tests..." instead of "Discovers tests..." in the first sentence describing a method).
```